values/dev/network/wallaby-archive.yaml

```diff
@@ -15,6 +15,7 @@ Lotus:
   env:
     default:
       INFRA_CLEAR_RESTART: "true"
+#      INFRA_PERSISTNODEID: "true"
```

Comment:
Please remove this extra line :)

Reply:
It just hot fix.
I still proceed working on INFRA_PERSISTNODEID for wallaby